toml.md

```diff
@@ -797,23 +797,32 @@ Inline Table
 ------------
 
 Inline tables provide a more compact syntax for expressing tables. They are
-especially useful for grouped data that can otherwise quickly become verbose.
-Inline tables are fully defined within curly braces: `{` and `}`. Within the
-braces, zero or more comma-separated key/value pairs may appear. Key/value pairs
-take the same form as key/value pairs in standard tables. All value types are
+especially useful for grouped (deeply) nested data that can otherwise quickly
+become verbose.
+
+Inline tables are defined within curly braces: `{` and `}`. Within the braces,
```

Comment:
```suggestion
Inline tables are fully defined within curly braces: `{` and `}`. Within the braces,
```

The fully here is important, since it notes that you can't "inject" values into it.

Reply:
Alrighty; I updated the PR.

Reply:
Oh, wow, thanks for the super quick response on this! :)

Reply:
Happened to be on GitHub waiting for the CI to finish on another project :-)